index.bs

```diff
@@ -11576,13 +11576,13 @@ Background</h3>
 
 A common feature requirement for modern 3D games is the ability to
 dynamically spatialize and move multiple audio sources in 3D space.
-Game audio engines such as OpenAL, FMOD, Creative's EAX, Microsoft's
-XACT Audio, etc. have this ability.
+Game audio engines such as <a href="https://openal.org">OpenAL</a>, <a href="http://www.fmod.org">FMOD</a>, <a href="https://www.creative.com/oem/technology/eax.asp">Creative's EAX</a>, <a href="https://www.creative.com/oem/technology/eax.asp">Microsoft's
```

Comment:
Not really sure if we should provide links here.  And I'm not sure I got them all right.  And XACT Audio has reached end-of-life, so maybe it should be removed completely.

Reply:
Well, I am not sure why we have to provide those specific names in here. I am fine with removing all of them.

Reply:
WFM.  I'll just remove this.

Reply:
Per offline discussion, we'll remove everything but leave OpenAL since that was used as the basis for much of the `PannerNode`.